nixos/modules/system/boot/kernel.nix

```diff
@@ -211,21 +206,11 @@ in
         "xhci_hcd"
         "xhci_pci"
         "usbhid"
-        "hid_generic" "hid_lenovo"
-        "hid_apple" "hid_logitech_dj" "hid_lenovo_tpkbd" "hid_roccat"
+        "hid_generic" "hid_lenovo" "hid_apple" "hid_roccat"
 
         # Misc. keyboard stuff.
         "pcips2" "atkbd" "i8042"
```

Comment:
Maybe `i8042` could be removed from here assuming the kernel problem has been fixed? Not sure if that's worth it though.

Reply:
I've investigated a bit but wasn't able to find any information. It's safer to keep it around for now IMHO.